Group7.java

```diff
@@ -0,0 +1,9 @@
+public class Group7 {
+
+    //Khalil
+    //Mohammed
+    //Ahmed
+    //Senait
+    //Naseem
```

Comment:
you should remove one member. If you review @Qasaaaye's direction carefully, he requested you to add a member which means not more than one. 
Thank you

Reply:
yes, please remove 1